Browser/base/librarycomponent.py

```diff
@@ -48,6 +48,10 @@ def __init__(self, library: "Browser") -> None:
     def playwright(self):
         return self.library.playwright
 
+    @property
+    def auto_closing_level(self) -> AutoClosingLevel:
+        return self.library._auto_closing_level
+
```

Comment:
Why is in playwright.py this is used and at the other situations the _ one?

Reply:
Need more words?

Reply:
Now I have understood, changed